src/board/game.mjs

```diff
@@ -0,0 +1,31 @@
+// functions that handle the evaluations of the game
```

Comment:
i would maybe name this file `game-evaluations` if that is what the functions in this file do